### PR TITLE
Added Unified Customer Profile Scenario

### PR DESCRIPTION
**Pull request by marvinbuss**
## Summary of the Pull Request
Added Unified Customer Profile Scenario

## PR Checklist
* [x] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md).

## Validation Steps Performed
See checks below

### CODE REVIEW

File: fsi/solutions/bankingCustomerEngagement/readme.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ Specifically for the banking customer engagement scenario, you need:
     * Omnichannel for Customer Service
     * Customer Insights (optional but required for full experience)
 
-> Note: The implementation of the Banking Customer Engagement solution can currently not be automated, and requires manual implementation and configuration across Solution Center, Power Platform Admin Center.
+> Note: The implementation of the Banking Customer Engagement solution can currently not be automated, and requires manual implementation and configuration across Solution Center and Power Platform Admin Center.
```
Comment (amanjeetsingh) [inline fsi/solutions/bankingCustomerEngagement/readme.md:32]: Consider changing "... requires manual implementation" to either ".. requires manual inputs across Solution Center and Power ..."

?

File: fsi/solutions/unifiedCustomerProfile/readme.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Prerequisites
+
+Before you deploy and configure, verify you have implemented the [prerequisites](../../prereqs.md).
+
+Specifically for the unified customer profile scenario, you need:
+
+* [Power Platform environments](../../../foundations/powerPlatform/)
+  * Must be created upfront, in United States with Dataverse and D365 Apps enabled
+
+> Note: The implementation of the unified customer profile solution can currently not be automated, and requires manual implementation and configuration across Solution Center and Power Platform Admin Center.
```
Comment (amanjeetsingh) [inline fsi/solutions/unifiedCustomerProfile/readme.md:24]: Please change to ".... customer profile solution cannot be fully-automated and requires manual intervention to complete configuration steps across Solution Center and Power Platform Admin Center."

Thanks

File: fsi/solutions/unifiedCustomerProfile/readme.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+The financial holdings tab provides more detailed insights about the financial holdings shown in the summary tab. Following information can be viewed in this section:
+
+* *Financial holdings*: Financial holdings name, category, type, value, instruments, alerts for savings or loans nearing the maturity date
+* *Currency treatment*: Detailed fianncial holdings overview in the original currency and teh default currency
+
+## Planning guidelines for unified customer profile
+
+This section provides prescriptive guidance with design considerations and recommendations across the critical design areas for unified customer profile for the teams that will deploy and manage the capabilities within the Microsoft Cloud for Financial Services.
+
+> Note: The design areas for unified customer profile are lightweight as the critical design areas, considerations and recommendations should be addressed in the Power Platform architecture itself. This is primarily because the unified customer profile scenario and its dependencies are D365 applications that must only be sequenced and deployed into existing environments.
```
Comment (amanjeetsingh) [inline fsi/solutions/unifiedCustomerProfile/readme.md:76]: Consider rewording? 

"United customer profile is based on Dynamics 365 and Power Platform, hence it inherits design decisions and implementation details from the underlying foundational building blocks."

**Review by amanjeetsingh — COMMENTED** [review]
Proposed minor changes.

File: fsi/solutions/bankingCustomerEngagement/readme.md (hunk 1 above)
Comment (marvinbuss) [reply fsi/solutions/bankingCustomerEngagement/readme.md:32]: This is currently the consistent wording for all docs. I would prefer to keep it consistent here as well. What do you think?

File: fsi/solutions/unifiedCustomerProfile/readme.md (hunk 3 above)
Comment (marvinbuss) [reply fsi/solutions/unifiedCustomerProfile/readme.md:76]: Same a above

File: fsi/solutions/unifiedCustomerProfile/readme.md (hunk 2 above)
Comment (amanjeetsingh) [reply fsi/solutions/unifiedCustomerProfile/readme.md:24]: If it's the same across other documents, I think we should consider making it more concise in future. 
Maybe I am being pedantic. Let's chat offline. Approving for now.

File: fsi/solutions/unifiedCustomerProfile/readme.md (hunk 3 above)
Comment (amanjeetsingh) [reply fsi/solutions/unifiedCustomerProfile/readme.md:76]: Same as above - let's take it offline. I think we can make verbiage more concise. 
Maybe I am just being pednatic :)

**Review by amanjeetsingh — APPROVED** [review]

**Pull request merged**